src/components/Transactions.tsx

```diff
@@ -282,15 +301,30 @@ export const Transactions: FC<TransactionsProps> = () => {
                     ))
                   : ''}
               </Box>
+              <Divider />
+              <Box sx={{ mt: 2 }}>
+                {activeCategory
+                  ? activeCategory.map((c:any) => {
```

Comment:
why active category array? do we have situations when we have two active categories?